run_dir/static/js/libpooling_queues.js

```diff
@@ -137,18 +145,6 @@ $('body').on('click', '.group', function(event) {
 });
 
 
-// Copy project samples table to clipboard
-var clipboard = new Clipboard('#libpools_copy_table');
-clipboard.on('success', function(e) {
-  e.clearSelection();
-  $('#libpools_copy_table_btn').addClass('active').html('<span class="fa fa-copy"></span> Copied!');
-  setTimeout(function(){
-    $('#libpools_copy_table_btn').removeClass('active').html('<span class="fa fa-copy"></span> Copy table');
-  }, 2000);
-});
-
-$('#libpools_copy_table_btn').on('click', function () {
-  $('.expand-all').click();
-  $('#libpools_copy_table').click();
+$('.dt-buttons > .buttons-copy').on('click', function () {
```

Comment:
Is it required to expand everything when the copy button is clicked?

Reply:
I don't know, you created the page- is it? 😆 

I'm guessing you're saying "no", ok, I'll remove it!

Reply:
I don't know with the new one, the old solution needed it.

Reply:
Have you checked with the new buttons?

Reply:
You mean if it's possible to expand after the copy button is clicked? OK, I wouldn't be able to test it since there isn't any data locally 🤔

Reply:
I haven't added it to qPCR queues, I'll add it and deploy to stage, because that data is available!